wolox-books/src/components/BooksList.vue

```diff
@@ -19,9 +17,8 @@ export default {
   },
   methods: {
     getBooks() {
-      getListOfBooks().then(response => {
-        this.booksToShow = response.data
-      })
+      this.$store.dispatch('bringBooks')
+      this.booksToShow = this.$store.getters.allBooks
```

Comment:
I think this is way more clear with the `mapActions` and `mapGetters` helpers from `vuex`. Check them out please :smile: